jsconfig.json

```diff
@@ -1,6 +0,0 @@
-{
```

Comment:
Can we leave this file in here? VS Code uses it (or at least used to) for JS, and we will still have some JS files (gulp tasks, etc.)